lib/internal/validators.js

```diff
@@ -34,10 +34,18 @@ const {
 } = require('internal/util/types');
 const { signals } = internalBinding('constants').os;
 
+/**
+ * @param {*} value
+ * @returns {boolean}
```

Comment:
```suggestion
 * @returns {value is number}
```

Can we also do that here?

Reply:
It would return false positives, wouldn't it?

Reply:
Yes!

Reply:
You're right, `isInt32(0.5)` returns `false` even though it's a number (false negative?).
Ideally it would narrow the type only in the positive case but that's probably not doable.

Reply:
I see. I'll remove it!

Reply:
Fixed with a force push to the last commit.